### ISSUE TITLE
OpenShot-v2.4.3-x86_64.AppImage: stuck in endless loop and cannot recover

### ISSUE THREAD

**Issue by foxy49** [OpenShot/openshot-qt#2485]
After a 'freeze' while creating a slideshow with around 30 jpg files with transition effects,  I was unable to close Open Shot and restarted my Linux Mint 18.3 Cinammon system.

After the reboot,  when I tried to open Open Shot, there was each time a momentary 'flash' on the screen caused by the program opening and immediately closing.

To overcome this I deleted my project files (the jpg images).  

Now the program opens but is stuck in an endless loop asking me for the location of the missing files;  I cannot access menu options or cancel this and I cannot even close the program again.

How do I escape from this loop so I can start with no automatic search for a missing project?

**Comment by peanutbutterandcrackers** [comment]
For now, perhaps restart your computer, and then add the [daily PPA](https://www.openshot.org/ppa/) and then install OpenShot from the terminal using `sudo apt install openshot-qt`. Once that is done, try re-opening the project and point the prompt to the directory containing your jpg files.

The reason you have to add the daily ppa instead of the stable one is that the fix for this has only recently gone into the source and will be released in the next version. It is my understanding that you can't wait that long.

On another note, perhaps [this](https://youtu.be/VE6awGSr22Q?list=RDQM2Gisq8_E2Hc&t=254) would be the more easier way to create such a slideshow.

Hope that helps.

Closing as answered.

**Comment by foxy49** [comment]
Thanks for the advice. However I installed the latest version of Openshot-qt at the terminal and executed it.  The problem is the same.  It is looking for the 'missing' files of my aborted project.  If I direct the program to the missing files, it immediately aborts.

In previous experimenting with this, if I place all the files except for one in the directory it expects, it will open, inform me about that one missing file, and if I access it then the program closes instantly as previously.

Is there any way to prevent Open Shot looking on opening for the previous project?

**Comment by foxy49** [comment]
I have seen this problem described and apparently solved for WINDOWS with the suggestion:  "You need to either recreate a file with the same name or delete your OpenShot preferences folder. ".openshot_qt"   (This makes it appear to Open Shot that it is opening for the first time).

What is the equivalent file in Linux and where would I find it?

**Comment by foxy49** [comment]
The solution is of course very easy but nobody suggested it to me so it's probably worth mentioning.

I downloaded Open Shot on to a second computer.  I created a very simple project with source files and osp file saved to the same /home/temp directory.

I copied these files and directory to my laptop with the problem.

I opened Open Shot by clicking on the saved osp file and clicking 'open with Open Shot'.

I then carefully quit the file and tried making another simple project file (belt and braces).

Now Open Shot opens as normal and has stopped looking for the missing files and the 'broken' project.

**Comment by peanutbutterandcrackers** [comment]
@foxy49 - Does that men deleting `.openshot_qt` didn't fix it for you? Perhaps this issue might need some investigations... Hmm... :confused:

**Comment by BertieFox** [comment]
I found and deleted /usr/share/openshot-qt/openshot_qt but that made no difference in my case.

What was bizarre was that Open Shot would go through all the 'normal' requests for missing files and open if just one was missing (any one, not just a particular one),  but as soon as I directed it to the last missing file,  the program instantly closed. 

I remember I was messing with a whole lot of transitions when the problem first occurred causing the freeze and subsequent problem in getting the program to open.  All seems to be working fine now but if I manage to replicate the problem I will let you know and note carefully the circumstances.

Thank you for your advice and help.
